geopandas_view/test_view.py

```diff
@@ -1,15 +1,208 @@
 import geopandas as gpd
+import pandas as pd
 from geopandas_view import view
 
+from .view import _BRANCA_COLORS
+
 nybb = gpd.read_file(gpd.datasets.get_path("nybb"))
 world = gpd.read_file(gpd.datasets.get_path("naturalearth_lowres"))
 cities = gpd.read_file(gpd.datasets.get_path("naturalearth_lowres"))
 
 
-def test_simple():
+def test_simple_pass():
+    """Make sure default pass"""
     m = view(nybb)
     m = view(world)
     m = view(cities)
+    m = view(world.geometry)
+
+
+def test_choropleth_pass():
+    """Make sure default choropleth pass"""
+    m = view(world, column="pop_est")
+
+
+def test_map_settings_default():
+    """Check default map settings"""
+    m = view(world)
+    assert m.location == [-3.1774349999999956, 2.842170943040401e-14]
+    assert m.options["zoom"] == 10
+    assert m.options["zoomControl"] == True
+    assert m.position == "relative"
+    assert m.height == (100.0, "%")
+    assert m.width == (100.0, "%")
+    assert m.left == (0, "%")
+    assert m.top == (0, "%")
+    assert m.global_switches.no_touch is False
+    assert m.global_switches.disable_3d is False
+    assert "openstreetmap" in m.to_dict()["children"].keys()
+
+
+def test_map_settings_custom():
+    """Check custom map settins"""
+    m = view(nybb, zoom_control=False, width=200, height=200, tiles="CartoDB positron")
+    assert m.location == [40.70582377450201, -73.9778006856748]
```

Comment:
When I tried this locally, this location for me was `[40.70582408558524, -73.97780021351167]` not sure why maybe  I could add `math.isclose` here?

Reply:
That is quite a big difference considering that it is the mean of total bounds. Weird. I'll add `pytest.approx` in here.